### DISCUSSION TITLE
Lesson 7 importing Test from forge-std

### DISCUSSION THREAD

**Discussion by Abhishek-2416** [Cyfrin/foundry-full-course-cu#971] [Q&A] [answered]
Although all the files exists at the right place 
Still i wind up with this error Source "forge-std/Test.sol" not found: File import callback not supported(6275)
This is the second time , already tried doing it again from the start

**Comment by olahamid** [comment]
> Although all the files exists at the right place Still i wind up with this error Source "forge-std/Test.sol" not found: File import callback not supported(6275) This is the second time , already tried doing it again from the start

you need to make sure you are importing your forge std correctly. try copy the path of forge std and paste i in your import line

**Comment by Abhishek-2416** [comment]
I did that too
Now for context i was using vscodium idk what wrong with it 
still shows me that error it got resolved only when i restarted it

**Reply by olahamid** [reply, marked as answer]
> I did that too
> Now for context i was using vscodium idk what wrong with it 
> still shows me that error it got resolved only when i restarted it
> 

 I don't use vscodium at all. I use vs code and this what I think the issue might be.
 i think the issue might be with the extension, try using a different extension(change to nomic foundation) and follow the normal "../lib/forge-std/src/...." way.

**Reply by Abhishek-2416** [reply]
Yeah i did that too but still couldntt get it to work
It got resolved only when i restarted vscodium 
Not sure whats the issue with vscodium but this keeps happening
